config/database.js

```diff
@@ -1,17 +1,17 @@
+// mongoose connection
+const mongoose = require('mongoose');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
const mongoose = require("mongoose");
```